plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -406,7 +406,14 @@ func (c *CiscoTelemetryMDT) handleTelemetry(data []byte) {
 
 		// Parse values
 		for _, subfield := range content.Fields {
-			c.parseContentField(grouper, subfield, "", msg.EncodingPath, tags, timestamp)
+			prefix := ""
+			if subfield.Name == "operation-metric" {
+				prefix = subfield.Fields[0].Fields[0].GetStringValue()
+			}
+			if subfield.Name == "class-stats" {
+				prefix = subfield.Fields[0].Fields[1].GetStringValue()
+			}
```

Comment:
Can we please make this a `switch` statement in case more "names" are relevant later?
```suggestion
			switch subfield.Name {
			case "operation-metric":
				prefix = subfield.Fields[0].Fields[0].GetStringValue()
			case "class-stats":
				prefix = subfield.Fields[0].Fields[1].GetStringValue()
			}
```

Reply:
@srebhan it makes sense, I applied the suggested change